wireless_modules/back_module/main.py

```diff
@@ -13,24 +15,36 @@
 # RZERO (for calibration of MQ135) found when the sensor was first 'activated'
 RZERO = 8.62
 
+# Circumference of a 700x28 bike wheel in meters.
+WHEEL_CIRCUMFERENCE = 2.136
+
+# Define Voltage divider factor for this module or leave as None to use default voltage factor
+VOLTAGE_FACTOR = None
+
 
 async def main():
     # Define all the Pin objects for each sensor
     mq135_pin = machine.Pin(34)
+    reed_pin = machine.Pin(5)
 
     # Instantiate sensor objects
     my_mq135 = CO2(mq135_pin)
     my_mq135.set_rzero(RZERO)
 
     my_gps = GpsSensor(2)
 
+    my_reed = ReedSensor(reed_pin, WHEEL_CIRCUMFERENCE)
+
+    battery_pin = 33
+    battery_reader = BatteryReader(battery_pin, scale=1, voltage_factor=VOLTAGE_FACTOR)
+
     # Set up the wireless module
-    back_module = WirelessModule(MODULE_NUM)
-    sensors = [my_mq135, my_gps]
+    back_module = WirelessModule(MODULE_NUM, battery_reader)
+    sensors = [my_mq135, my_gps, my_reed]
     back_module.add_sensors(sensors)
 
     print("Starting asyncio loop")
-    asyncio.create_task(back_module.run())
+    asyncio.create_task(back_module.run(battery_data_rate=1))
```

Comment:
Do we want to change this to our intended `300` seconds instead.

Reply:
Damnit that's me testing everythings working and not reverting changes again

Reply:
Thanks for being such a diligent reviewer, fixed now